cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller_test.go

```diff
@@ -991,3 +992,97 @@ func TestControllerMachineSetNodeNamesUsingStatusNodeRefName(t *testing.T) {
 		}
 	}
 }
+
+func TestControllerGetAPIVersionGroup(t *testing.T) {
+	expected := "myversion.mygroup"
+	err := os.Setenv(machineAPIVersionGroupEnvVar, expected)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	observed := getAPIVersionGroup()
+	if observed != expected {
+		t.Fatalf("Wrong Version Group detected, expected %v, got %v", expected, observed)
```

Comment:
Nit: we know expected and observed are strings, could use `%q` instead of `%v` (This applies on several lines through this PR) to quote the string and make the error more obvious maybe?
```suggestion
		t.Fatalf("Wrong Version Group detected, expected %q, got %q", expected, observed)
```